packages/backend/src/Metrics.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  collectDefaultMetrics,
+  Counter,
+  CounterConfiguration,
+  Gauge,
+  GaugeConfiguration,
+  Histogram,
+  HistogramConfiguration,
+  register,
+  Summary,
+  SummaryConfiguration,
+} from 'prom-client'
+
+export class Metrics {
```

Comment:
Why is this a class? You are neither holding any state inside it, nor injecting other services inside it?  Do you plan to do it in the future? If not, maybe a single export would be better?
 ```
export const Metrics = {
  init,
  getMetrics
}
```

Reply:
The idea behind it being a class is that 

1. it allows us to change the client library easily in the future
2. it's trivial to mock for tests